README.md

```diff
@@ -52,7 +52,7 @@ For command line help, visit our wiki page on Bourbon’s [command line interfac
 
   It’s not recommended to add or modify the Bourbon files so that you can update them easily.
```

Comment:
> It’s not recommended to add or modify the Bourbon files so that you can update them easily.

What does that mean? Is this left over from when `bourbon install` was a thing?